tests/core/test_parsing.py

```diff
@@ -271,13 +271,18 @@ def test_no_DOB_field_when_DOB_not_provided():
     assert not parsed_content.get(Fields.DOB)
 
 
-@pytest.mark.parametrize('page,start,end', (('traffic-fatality-50-3', 'Cedric', '| 01/26/1992'), ))
-def test_extract_deceased_field_twitter(page, start, end):
+@pytest.mark.parametrize('page,before,after', (('traffic-fatality-50-3', '1982', 'Aa'), ))
+def test_extract_deceased_field_twitter(page, before, after):
+    """
+    Ensure that parsing of the Twitter fields results in two different
+    records about deceased people, and that one ends and the next begins
+    in the right place.
+    """
     page_text = load_test_page(page)
     parsed_content = parsing.parse_twitter_fields(page_text)
     deceased = parsed_content[Fields.DECEASED]
-    assert deceased[0].startswith(start)
-    assert deceased[-1].endswith(end)
+    assert deceased[0].strip().endswith(before)
+    assert deceased[-1].strip().startswith(after)
```

Comment:
Way better docstring 👍 

However I still need a bit more information regarding how to use the `twitter_deceased_field_to_list` function (this is where I was expecting an improved docstring 😉 ).

I want to pin the test to the exact function which parses the values:
```python
def test_extract_deceased_field_twitter(raw_deceased, start, end):
    deceased = parsing.twitter_deceased_field_to_list(raw_deceased)
    assert deceased[0].startswith(start)
    assert deceased[-1].endswith(end)
```

But if I provide this value as an input for instance:
```python
description_04 = (
    "Case:          19-2291933 Date:           Saturday, August 17, 2019 Time:          10:20 p.m. "
    "Location:    4500 FM 2222/Mount Bonnell Road    Deceased 1:  Cedric Benson | Black male | 12/28/1982 "
    "Deceased 2:  Aamna Najam | Asian female | 01/26/1992"
```

The test fails with the following error:
```bash
E       AssertionError: assert False
E        +  where False = <built-in method startswith of str object at 0x103fcc930>('Cedric')
E        +    where <built-in method startswith of str object at 0x103fcc930> = 'Case:          19-2291933 Date:           Saturday, August 17, 2019 Time:          10:20 p.m. Location:    4500 FM 2222/Mount Bonnell Road    '.startswith

tests/core/test_parsing.py:283: AssertionError
```

And the following returned values:
```python
>>> len(deceased)
3
>>> deceased[0]
'Case:          19-2291933 Date:           Saturday, August 17, 2019 Time:          10:20 p.m. Location:    4500 FM 2222/Mount Bonnell Road    '
>>> deceased[1]
' 1:  Cedric Benson | Black male | 12/28/1982 '
>>> deceased[2]
' 2:  Aamna Najam | Asian female | 01/26/1992'
>>>
```

So I need to understand how/where you got rid of the `1: ` and `2: ` in order to make the tests pass.

Reply:
I added a test for this. The function does little more than split a string on the word Deceased. I probably wouldn't have made it its own function if I didn't have to to pass flake8.

The function is only supposed to take text that comes after the first instance of the word Deceased, because that's what the other functions would have stored as `d["Deceased"]`. It split your test string into 3 instead of 2 because your test string included both instances of "Deceased".